extensions/google-translate/package.json

```diff
@@ -83,7 +83,7 @@
           "value": "zh-CN"
         },
         {
-          "title": "🇨🇳 Chinese Traditional",
+          "title": "🇹🇼 Taiwan Mandarin",
```

Comment:
Taiwanese maybe?

Reply:
Thank you for approving PR. Yes, you're right, let me update it. 🙏